README.md

```diff
@@ -4,7 +4,9 @@ This is a plugin for a local DynamoDB instance using Docker.
 
 When you are using Architect's sandbox mode, the plugin starts a Docker container running the [`amazon/dynamodb-local:latest`](https://hub.docker.com/r/amazon/dynamodb-local) image.
 
-This can be used alongside other plugins such as [@hicksy/arc-plugin-sandbox-stream](https://github.com/hicksy/arc-plugin-sandbox-stream) to allow for more realistic debugging with `tables-streams`.
+If you have defined a `@tables-streams` section in your `app.arc` file, the respective streams will be enabled.
+
+There is an issue with the Docker image of DynamoDB local that causes a `TrimmedDataAccessException` error to be thrown on the first read. This problem lies deeper within DynamoDB and is out of the scope of this plugin. To handle this issue, there is a reset function that will trigger automatically. In practice, this means that the first invocation will fail (you will see a logged `TrimmedDataAccessException` message in your console), but the following invocations will work successfully.
```

Comment:
Is there an issue or AWS help page that describes the issue?

Reply:
Still looking, but so far no, just the other projects that I listed ^^ who also have run into the issue, and the doc pages on the error https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-dynamodb-streams/Class/TrimmedDataAccessException/ which is not particularly helpful

Reply:
Are you sure that you need to reset the stream when you get a TrimmedDataAccessException? It won't eventually start giving you fresh data once you have advanced beyond the trimmed data?

Reply:
Yes, since there is no way to advance if the TrimmedDataAccessException is thrown other than just doing a fresh request for a new shard iterator

Reply:
> Still looking, but so far no, just the other projects that I listed ^^ who also have run into the issue, and the doc pages on the error https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-dynamodb-streams/Class/TrimmedDataAccessException/ which is not particularly helpful

By "other projects," are you referring to https://github.com/hicksy/arc-plugin-sandbox-stream? I don't see any reference to TrimmedDataAccessException in that project.